omod/src/main/java/org/openmrs/web/controller/maintenance/CurrentUsersController.java

```diff
@@ -30,26 +29,18 @@
  * @see LoginServlet
  * @see org.openmrs.web.SessionListener
  */
-@Controller
-public class CurrentUsersController {
+public class CurrentUsersController extends SimpleFormController {
 	
 	protected final Log log = LogFactory.getLog(getClass());
 	
 	/**
-	 * Lists current users.
-	 * 
-	 * @param request
-	 * @param modelMap
+	 * @see org.springframework.web.servlet.mvc.AbstractFormController#formBackingObject(javax.servlet.http.HttpServletRequest)
 	 */
-	@RequestMapping(method = RequestMethod.GET, value = "admin/maintenance/currentUsers.list")
-	public String listCurrentUsers(HttpServletRequest request, ModelMap modelMap) {
+	protected List<String> formBackingObject(HttpServletRequest request) throws ServletException {
 		log.debug("List current users");
 		if (!Context.hasPrivilege(PrivilegeConstants.GET_USERS)) {
 			throw new APIAuthenticationException("Privilege required: " + PrivilegeConstants.GET_USERS);
 		}
-		
-		modelMap.put("currentUsers", CurrentUsers.getCurrentUsernames(request.getSession()));
-		return "/module/legacyui/admin/maintenance/currentUsers";
+		return CurrentUsers.getCurrentUsernames(request.getSession());
 	}
-	
```

Comment:
Because this existing controller is doing so little, I reverted it to an xml-based controller, which allows me to override this URL in my module by doing so in an annotated controller.  This page doesn't actually work anymore, but that's beside the point.